src/goto-programs/adjust_float_expressions.cpp

```diff
@@ -0,0 +1,200 @@
+/*******************************************************************\
+
+Module: Symbolic Execution
+
+Author: Daniel Kroening, kroening@kroening.com
+
+\*******************************************************************/
+
+/// \file
+/// Symbolic Execution
+
+#include "adjust_float_expressions.h"
+#include "goto_model.h"
+
+#include <util/cprover_prefix.h>
+#include <util/expr_util.h>
+#include <util/std_expr.h>
+#include <util/symbol.h>
+#include <util/ieee_float.h>
+
+#include <util/arith_tools.h>
+
+static bool
+have_to_adjust_float_expressions(const exprt &expr, const namespacet &ns)
+{
+  if(
+    expr.id() == ID_floatbv_plus || expr.id() == ID_floatbv_minus ||
+    expr.id() == ID_floatbv_mult || expr.id() == ID_floatbv_div ||
+    expr.id() == ID_floatbv_div || expr.id() == ID_floatbv_rem ||
+    expr.id() == ID_floatbv_typecast)
```

Comment:
I would strongly encourage a plain move without reformatting.

Reply:
The clang-format stage is marked as `allow_failures`, so you don't actually have to make it succeed.